ts/components/wallet/card/ActionIcon.tsx

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
This seem a generic component. Isn't better to put it outside the waller subdirectory?

Reply:
It is but it would overlap in functionality with the component defined in #222, so I would treat ActionIcon as a component I will dispose of when that PR is approved.